docs/trace/exception-handling/README.md

```diff
@@ -0,0 +1,70 @@
+# Exception Handling
+
+## First Chance Exception
+
+The term `First Chance Exception` is used to describe exceptions that are
+handled by the application - whether in the user code or the framework.
```

Comment:
`handled` here may be a little vague, as there is first chance for either handled or unhandled exception for the debugger. Perhaps borrow definition from below link?

https://devblogs.microsoft.com/devops/understanding-exceptions-while-debugging-with-visual-studio

Reply:
I've updated the doc to avoid first/second chance since these are more from debugger's perspective. I've borrowed the idea of using user-handled vs. unhandled exceptions.